src/encode_task_pool_ta.py

```diff
@@ -53,7 +53,7 @@ def pool_ta(tas, col, basename_prefix, out_dir):
         run_shell_cmd(cmd)
         return pooled_ta
     else:
-        return make_hard_link(tas[0], out_dir)
+        raise Exception('Needs at least two TAs (or BEDs) to be pooled.')
```

Comment:
This should raise something more specific, in this situation I would say `ValueError` would be appropriate. Also are there tests for this?

Reply:
Yes that makes sense. I will change it to `ValueError`.

There is currently no unit test for this wrapper `.py`. There is a higher level (task) test for this in WDL level. But it's not possible to test error conditions since there is no `try-except` thing in WDL.

Reply:
Fixed as you suggested.